src/compounds/product-table/src/stories.tsx

```diff
@@ -48,6 +48,23 @@ export const ExampleWithKnobs = () => {
   const thirdCol = select('Third column', columns, 'Content from knob')
   const fourthCol = select('Fourth column', columns, 'None')
 
+  console.log(
+    'Value',
+    ProductTable.text.Value({
+      value: 300,
+      unit: 'pounds',
+      subscript: '/ month'
+    })
+  )
+  console.log(
+    'Range',
+    ProductTable.text.Range({ from: 30, to: 260, unit: 'pounds' })
+  )
+  console.log(
+    'Subscript',
+    ProductTable.text.TextSubscript({ text: '£121', subscript: '/ month' })
+  )
+
```

Comment:
Added the console logs for testing - Happy to remove but useful to see the formatters working